### DISCUSSION TITLE
any timeout figure?

### DISCUSSION THREAD

**Discussion by xuze1993** [coder/code-server#2371] [Q&A]
I deployed the code-server on a public ip, with no-auth,I got a blank webpage with my laptop all the way.

But I found I can access the web on the  machine which code-server is on,It seems something timeout? Is there any variables?

**Comment by code-asher** [comment]
Could you post the browser logs?

Usually a blank page means that the web sockets aren't connecting. Are
you accessing code-server directly or through a proxy?
